src/components/fin/metricSlider/heading/HeadingButtons.tsx

```diff
@@ -45,9 +43,15 @@ export default function HeadingButtons({ size = 'md', endPage }: Props) {
 
   const timeoutRef = useRef<NodeJS.Timeout | null>(null)
 
+  // Detect with userAgent
+  const isMobile =
+    /Mobile|webOS|BlackBerry|IEMobile|MeeGo|mini|Fennec|Android|iP(ad|od|hone)/i.test(
+      navigator.userAgent
+    )
+
   const handleShare = async () => {
     // Desktop : only copy the url
-    if (!navigator?.share || window.innerWidth > MAX_WIDTH_MOBILE) {
```

Comment:
Don't you need to remove `MAX_WIDTH_MOBILE` ?

Reply:
Yes !

Reply:
Oh it is already removed actually